tests/testthat/test-z-integration.R

```diff
@@ -384,16 +411,18 @@ test_that("can get report parameters", {
 })
 
 test_that("can get report parameters with no commit ID", {
-  path <- orderly_prepare_orderly_git_example()
-  ## Checkout a branch with a report with parameters
-  orderly_git_checkout_branch("other", root = path[["local"]])
-  server <- start_test_server(path[["local"]])
+  path <- orderly_prepare_orderly_example("interactive", testing = TRUE,
```

Comment:
This is a change of behaviour which is maybe unexpected due to changes in `git_latest_commit`. Previously this was relying on the fact that `/reports/<name>/parameters` endpoint if called without a commit would use the currently checked out version of the repo. It now defaults to using master branch. We probably weren't relying on the previous default behaviour? What default behaviour do you think we should use? I can make some further changes if previous made more sense, or maybe even just not support calling this endpoint without a parameter for commit id.

Reply:
I think we should use the new behaviour, and use the default branch - that way if someone has checked out a branch to experiment on the server, the API behaviour is consistent